scripts/update.sh

```diff
@@ -23,71 +23,71 @@ echo "::: The updating functionality for PiVPN scripts is temporarily disabled"
 echo "::: To keep the VPN (and the system) up to date, use 'apt update' and 'apt upgrade'"
 exit 0
 
```

Comment:
Spacing in this file indeed is very inconsistent, however, the way it is changed here doesn't make it better at all 😉. I'd suggest to do this in a dedicated PR and remove changes for this file from this commit/PR.